analyzers/tests/SonarAnalyzer.UnitTest/TestCases/SymbolicExecution/Roslyn/NullPointerDereference.cs

```diff
@@ -540,6 +540,20 @@ void ShortCircuit_Reachable()
                 o.ToString();   // Compliant for the same reason
             }
         }
+
+        public void Type_IsAssignableFrom()
```

Comment:
This is the only test in NullPointerDerefernce that is about `typeof`. Add one more test case here, that comes without any "IsAssignable" complications, like e.g.
```cs
Type type = null;
type = typeof(object);
type.ToString(); // Compliant
```